SerpentsHand/Plugin.cs

```diff
@@ -18,11 +18,11 @@ namespace SerpentsHand
 	SmodMinor = 0,
 	SmodRevision = 0
 	)]
-	public class Plugin : Smod2.Plugin
+	public class SHPlugin : Plugin
```

Comment:
Renamed `Plugin` to `SHPlugin`, because **why would you ever name your class like an existing class >:c **